docker-compose.yml

```diff
@@ -144,6 +144,103 @@ services:
       - go-modules:/root/go/pkg/mod
       - root-cache:/root/.cache
 
+  # PMM with external Postgres DB
+  pg:
+    profiles:
+      - pmm-pg
+    build:
+      context: ./managed/testdata/pg
+      args:
+        POSTGRES_IMAGE: ${POSTGRES_IMAGE:-postgres:14}
+      dockerfile: Dockerfile
+    container_name: pmm-pg
+    environment:
+      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD:-pmm-password}
+    ports:
+      - ${POSTGRES_PORT:-5432}:5432
+    command: |
+      postgres 
+      -c shared_preload_libraries=pg_stat_statements 
+      -c pg_stat_statements.max=10000 
+      -c pg_stat_statements.track=all 
+      -c pg_stat_statements.save=off 
+      -c ssl=on 
+      -c ssl_ca_file=/certs/root.crt 
+      -c ssl_cert_file=/certs/server.crt 
+      -c ssl_key_file=/certs/server.key
+      -c hba_file=/conf/pg_hba.conf
+    networks:
+      - ${NETWORK:-default}
+  pmm-managed-server-pg:
```

Comment:
I think we don't need it, let's just migrate all required environment variables to the main one and comment

Reply:
moved to main pmm profile with commented env fields